app/controllers/events/routes.py

```diff
@@ -46,7 +45,9 @@ def signinEvent():
     """Utilizes form data and sign in function. Returns correct flasher message."""
     eventid = request.form["eventid"]
     bnumber = request.form["bNumber"]
-    programid = ProgramEvent.select(ProgramEvent.program).where(ProgramEvent.event == eventid)
+    
+    # Correct query
+    programid= Event.select(Event.program_id). where(Event.id == eventid)
```

Comment:
I don't think you tested this. A select query returns a ModelSelect object, not a program id.

Reply:
Thank you for the correction, Brian. Wasn't the previous code also returning a ModelSelect object? Should we use a .get() instead?